coordinator/src/main/scala/filodb.coordinator/queryplanner/PartitionLocationPlanner.scala

```diff
@@ -92,4 +92,43 @@ abstract class PartitionLocationPlanner(dataset: Dataset,
       case _: PromQlRemoteExec => false
       case _ => true
     }
+
+  /**
+   * Returns true iff the argument Aggregation can be pushed down.
+   * More specifically, this means that it is correct to materialize the entire plan
+   * with a lower-level planner and aggregate again across all returned plans.
+   *
+   * @param innerTschemaLabels occupied iff the inner plan can be pushed-down according to the set of labels.
+   */
+  protected def canPushdownAggregate(agg: Aggregate, innerTschemaLabels: Option[Seq[String]]): Boolean = {
```

Comment:
Wondering if we should add a config flag to enable this, default to true ( the logic looks good to me, but just in case for an unforeseen scenario ? ) feel free to close this as "won't fix"

Reply:
Good call -- I've added a PlannerParams attribute.